src/components/image/image.ts

```diff
@@ -269,7 +269,8 @@ export class SbbImageElement extends LitElement {
   /**
    * Whether to have no border-radius on the image
```

Comment:
Just a nitpick, this doc needs to be updated

Reply:
Thanks! fixed